presto-docs/src/main/sphinx/security/group-file.rst

```diff
@@ -0,0 +1,42 @@
+==========
+Group File
+==========
+
+Presto can map user names onto groups for easier access control and
+resource group management. Group file resolves group membership using
+a file on the coordinator.
+
+Group File Configuration
+------------------------
+
+Enable group file by creating an ``etc/group-provider.properties``
+file on the coordinator:
+
+.. code-block:: none
+
+    group-provider.name=file
+    file.group-file=/path/to/group.txt
+
+The following configuration properties are available:
+
+==================================== ==============================================
+Property                             Description
+==================================== ==============================================
+``file.group-file``                  Path of the group file.
+
+``file.refresh-period``              How often to reload the group file.
+                                     Defaults to ``1m``.
+==================================== ==============================================
+
+Group Files
+-----------
+
+File Format
+^^^^^^^^^^^
+
+The group file contains a list of groups and members, one per line,
+separated by a colon. Users are separated by a space.
+
+.. code-block:: none
+
+    group_name: user_1 user_2 user3
```

Comment:
Not to bikeshed on the format, but I'd expect this to use commas, like the UNIX `/etc/group` file format:
```
group_name:user_1,user_2,user_3
```
This would be the same format, minus the group ID and password fields.